src/common.js

```diff
@@ -123,9 +123,11 @@ function resolvePackages(pkgs) {
   for (var p in pkgs) {
     var value = pkgs[p];
     // TODO package fallback support
-    if (typeof value !== 'string')
-      continue;
-    outPkgs[resolveIfNotPlainOrUrl(p) || p] = value;
+    if (typeof value !== 'string') {
```

Comment:
This should rather be `if (Array.isArray(value))` with `else if (typeof value === 'string')`.